package.json

```diff
@@ -4,9 +4,17 @@
   "dependencies": {
     "grunt": "^1.0.1"
   },
+
+  "scripts": {
+	  "build": "grunt build",
+	  "serve": "grunt",
+	  "test": "grunt test"
+  },
+
   "devDependencies": {
     "grunt-contrib-concat": "^1.0.1",
     "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-watch": "^1.0.0"
+    "grunt-contrib-watch": "^1.0.0",
+	"grunt-cli": "^1.4.3"
```

Comment:
The formatting is a bit broken here (tabs are weird), a fix would be appreciated